pyflink-decodable/Makefile

```diff
@@ -0,0 +1,37 @@
+.PHONY: clean deploy
+
+TARGET = target
+LIBS = $(TARGET)/libs
+PYTHON_LIBS = $(TARGET)/python-libs
+
+PY = python3
+VENV = $(TARGET)/venv
+BIN=$(VENV)/bin
+
+build: $(TARGET) $(LIBS)/flink-sql-connector-kafka-3.0.2-1.18.jar $(LIBS)/flink-python-1.18.1.jar $(TARGET)/pyflink-job.zip $(VENV)
```

Comment:
Is this possibly out of date already? I attempted to build this and hit a 404 on the jar:

```
~/decodable/repos/examples/pyflink-decodable DE-6867
❯ make
mkdir target
mkdir -p target/libs
wget -N -P target/libs https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kafka/3.0.2-1.18/flink-sql-connector-kafka-3.0.2-1.18.jar
make: wget: No such file or directory
make: *** [target/libs/flink-sql-connector-kafka-3.0.2-1.18.jar] Error 1
```

Reply:
My bad, clearly I just like `wget`